pytorch_lightning/metrics/retrieval/retrieval_metric.py

```diff
@@ -0,0 +1,130 @@
+import torch
+from typing import List, Optional, Callable, Any
+
+from pytorch_lightning.metrics import Metric
+
+from pytorch_lightning.metrics.utils import get_mini_groups
+
+
```

Comment:
```suggestion
#: todo some decribtion here
```
 pls explain what is it for...

Reply:
You mean a comment directly over the import statement?

Reply:
yes and if you use `#:` it will ve also included in docs